### ISSUE TITLE
Sidebars and edit area not loaded correctly for user with System Admin and Wiki Approval role

### ISSUE THREAD

**Issue by hatuan195** [frappe/wiki#127]
In our setup, clicking edit on a wiki page only ok for Administrator.
Any other users even with System Admin and Wiki Approver roles only see blank edit page without sidebars like follow image

bench version output:
education 0.0.1
erpnext 14.18.3
frappe 14.28.2
hrms 15.0.0-dev
wiki 0.0.1

Do we need to set other roles?

![image](https://user-images.githubusercontent.com/47517542/225706790-ec98b4a5-2a44-44b9-bb0a-12383df0c66e.png)

**Comment by BreadGenie** [comment]
This is weird. The sidebar on /edit-wiki doesn't depend on permissions or roles afaik. I tried with both website user and system admin with wiki approval role and it works fine.
This feels like wiki failed to fetch the root sidebar but that too doesn't depend on any roles.

I've created a [telegram group](https://t.me/frappewiki) for wiki support. Feel free to join the group and we can nail down this issue.
